modules/apps/main.go

```diff
@@ -24,9 +24,11 @@ package main
 
 import (
 	"encoding/json"
+	"errors"
 	"os"
 	"strings"
 
+	"github.com/Nanocloud/community/modules/apps/lib/apps"
```

Comment:
I think in previous pull requests we gave a name to userPkg imports.

Not sure though, can you confirm you checked that ?